boltdb_test.go

```diff
@@ -20,6 +20,8 @@ func TestBoltDBNewBoltDB(t *testing.T) {
 	db.Close()
 }
 
+//TODO: ADD TESTS!!!
```

Comment:
Is this still a TODO?

Reply:
yes, there is an issue for it. To keep the scope of this pr small I will handle it in subsequent PRs,